playbook/app/pb_kits/playbook/pb_timeline/docs/_timeline_with_children.html.erb

```diff
@@ -0,0 +1,43 @@
+<%= pb_rails("timeline", props: {orientation: "horizontal", show_date: true}) do %>
+    <%= pb_rails("timeline/item", props: { line_style: "solid"}) do |item| %>
+
+      <% item.date_area do %>
```

Comment:
specifying the `item.date_area` allows us to only update the date area 

you don't have to provide the node or detail area if you don't want to. 

This is needed because our node area has default functionally and it will use the icon circle kit it you dont't put anything

Reply:
After doing more research i found that github's design system uses the same pattern https://primer.style/components/action-menu/rails/alpha#examples